src/constants.ts

```diff
@@ -1,2 +1,6 @@
 export const FREE_PLAN_BYTES_SPACE = 2 * 1024 * 1024 * 1024;
 export const FREE_INDIVIDUAL_TIER = 'free_individual_tier';
+
+export enum PRODUCT_IDS {
+  B2B = 'prod_Q6CYRA6w8fKZWK',
```

Comment:
We can use the product metadata to determine if it is a workspace plan or not, rather than hardcoding IDs here, as it won't scale. An example of product metadata is the field `maxSpaceBytes`

Reply:
Thanks that's a much cleaner solution.